lib/wilbertils/authorization/oauth2.rb

```diff
@@ -52,6 +52,9 @@ def token_request_headers
         case @@params[:grant_type].to_sym
         when :client_credentials
           { Authorization: "Basic #{Base64.strict_encode64("#{@@params[:body][:client_id]}:#{@@params[:body][:client_secret]}")}" }
+        when :password_credentials
+          {"Content-Type": 'application/x-www-form-urlencoded',
```

Comment:
instead of assuming the content type I think we should send that as part of headers so it'll be merged automatically at line 60